.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+# Auto-update GitHub Actions
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
I think that we need to include "pip", "Bundler", and "npm" here.  But most drafts won't include the necessary manifests, so I need to investigate whether the bot fails in that case.